forge-gui/res/cardsfolder/upcoming/necrodominance.txt

```diff
@@ -0,0 +1,13 @@
+Name:Necrodominance 
+ManaCost:B B B
+Types:Legendary Enchantment
+R:Event$ BeginPhase | ActiveZones$ Battlefield | ValidPlayer$ You | Phase$ Draw | Skip$ True | Description$ Skip your draw step.
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigChoose | TriggerDescription$ At the beginning of your end step, you may pay any amount of life. If you do, draw that many cards.
+SVar:TrigChoose:DB$ ChooseNumber | Max$ LifeTotal | AILogic$ Vermin | ListTitle$ Choose any amount of life to pay | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | NumCards$ X
```

Comment:
where's the payment?

Reply:
imo the previous version should have worked if you don't use ChooseNumber to overcomplicate it